### PR TITLE
Rework `ButtonTextLink*` to match requested framing

### LINKED ISSUE #472: Update ButtonLinks design/sizing
obrazek

### PR DESCRIPTION
**Pull request by hrach**
closes #472

### CODE REVIEW

**Review by Drekorian — APPROVED** [review]

File: ui/src/androidMain/kotlin/kiwi/orbit/compose/ui/controls/ButtonLink.kt [hunk 1]
```diff
@@ -35,10 +36,10 @@ public fun ButtonLinkPrimary(
     content: @Composable RowScope.() -> Unit,
 ) {
     ButtonLargePrimitive(
+        modifier = modifier.fillMaxWidth(),
```
Comment (gerak-cz) [inline ui/src/androidMain/kotlin/kiwi/orbit/compose/ui/controls/ButtonLink.kt:39]: Do we want to force this in buttons?

File: ui/src/test/snapshots/images/kiwi.orbit.compose.ui_ScreenshotTest_card.png
Comment (gerak-cz) [inline ui/src/test/snapshots/images/kiwi.orbit.compose.ui_ScreenshotTest_card.png:1]: Is this change intended?

Comment (hrach) [reply ui/src/test/snapshots/images/kiwi.orbit.compose.ui_ScreenshotTest_card.png:1]: Good question, I'll double-check. Generally speaking, yes, because the height of ButtonLink should need more space to fit in, the card with button link should have a different size.

File: ui/src/androidMain/kotlin/kiwi/orbit/compose/ui/controls/ButtonLink.kt (hunk 1 above)
Comment (hrach) [reply ui/src/androidMain/kotlin/kiwi/orbit/compose/ui/controls/ButtonLink.kt:39]: Very good question - I'm not fully sure. Those buttonlink buttons should be used in two cases - full-width ones with metrics of normal buttons and those "inner" ones - inside the text and also - that should have different metrics. So I made this one full-width by default. Usually, that's not the way in Compose, yet here it is probably the only preferred way. What do you think?

Comment (gerak-cz) [reply ui/src/androidMain/kotlin/kiwi/orbit/compose/ui/controls/ButtonLink.kt:39]: I agree it is the only preferred way. However, this has implications for the parent as well. So what if the parent wants to wrap onto the size of some other child (sibling of the button)? Wouldn't this be a problem?

File: ui/src/test/snapshots/images/kiwi.orbit.compose.ui_ScreenshotTest_card.png
Comment (hrach) [reply ui/src/test/snapshots/images/kiwi.orbit.compose.ui_ScreenshotTest_card.png:1]: obrazek

The designs aren't worth mentioning 😢 Title & button are not baseline aligned and the button "preserves" the right part just for itself. Therefore the bigger button's height doesn't shift the description down. 

I've put this for the next orbit meeting. Until them, waiting with this PR.

Comment (gerak-cz) [reply ui/src/test/snapshots/images/kiwi.orbit.compose.ui_ScreenshotTest_card.png:1]: Not sure I follow. The typical action uses the ButtonTextLink which should not use any more space than it needs, right?

**Comment by hrach** [comment]
Closing as the current approach seems to be a step back and Orbit is reconsidering the metrics.
